conformance/results/pyright/directives_deprecated.toml

```diff
@@ -0,0 +1,32 @@
+conformant = "Partial"
+notes = """
+Does not report error for deprecated magic methods
```

Comment:
Nit: for consistency, please add a period at the end of these sentences. These appear in the hover text within the summary report.